episodes/17-explanation.md

```diff
@@ -239,6 +239,18 @@ To check the formatting requirements, see the Introduction Episode example in yo
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+## Polishing a lesson
+
+You should not aim to make a polished lesson at the first attempt:
+keeping things relatively basic makes sense while you run through the first few iterations
```

Comment:
The suggestion that you shouldn't try to have a polished lesson sounds a bit odd to me.  Maybe better to reword that it is an unreasonable expectation to have a perfect lesson the first time you teach it.

```suggestion
Lessons do not need to be perfected prior to teaching them the first time.
In fact, keeping things relatively basic leaves you some flexibility while you run through the first few iterations
```